### ISSUE TITLE
New rule: autocomplite attribute has valid value

### ISSUE THREAD

**Issue by ajanec01** [w3c/wcag-act#489]
The ACT-Rules community group has completed work on [Autocomplete attribute has valid value](https://act-rules.github.io/rules/73f2c2). We would like to see this rule be added to the WCAG ruleset. You can find the rule here:

**Rule title**: Autocomplete has valid value
**Rule id**: 73f2c2
**Link to the rule**: https://act-rules.github.io/rules/73f2c2
**License**: ...  W3C Software And Document Notice And License 
**Implementors**:
- Axe-core
- Alfa
- QualWeb
- SortSite

If you'd like to contact me, please e-mail me at aron@diginclusion.com. (optional: You can also reach me on the W3C Community Slack, my name there is Aron Janecki)

**Comment by maryjom** [comment]
[Survey](https://www.w3.org/2002/09/wbs/93339/ACTAutocomplete/) open until 22 October.

**Comment by maryjom** [comment]
Being discussed in 12 November meeting.
